web/channels/user_socket.ex

```diff
@@ -19,8 +21,17 @@ defmodule Yggdrasil.UserSocket do
   #
   # See `Phoenix.Token` documentation for examples in
   # performing token verification on connect.
-  def connect(_params, socket) do
-    {:ok, socket}
+  def connect(%{"token" => token}, socket) do
+    # user id is stored in the token
+    case Phoenix.Token.verify(socket, "user socket", token, max_age: 1209600) do
```

Comment:
More of a note for my half awake self as to why the user_id was there in the first place, had a few different examples one storing and id one storing the user. haha.